### PR TITLE
add WPC and FEC configuration for SKU VSE2

### PR DESCRIPTION
**Pull request by novacain1**
This PR:

1. Adds XXVDA4T based ptp4l configuration for the OpenShift PTP Operator, responsible for acting as a Telecom Grandmaster (T-GM) to the group policy responsible for the VSE2 SKU in my lab.
2. Adds ACC100 based FEC acceleration policies to the group policy responsible for the VSE2 SKU in my lab.

### CODE REVIEW

File: clusters/ztp-policies/common/group-policies/group-smcx12-vse2.yaml [hunk 1]
```diff
@@ -37,7 +43,133 @@ spec:
...
+            ts2phc.master 1
+            [global]
+            use_syslog  0
+            verbose 1
+            logging_level 7
+            ts2phc.pulsewidth 100000000
+            #GNSS module s /dev/ttyGNSS* -al use _0
+            #cat /dev/ttyGNSS_1700_0 to find available serial port
+            #example value of gnss_serialport is /dev/ttyGNSS_1700_0
+            ts2phc.nmea_serialport /gpsd/data
```
Comment (jnunyez) [inline clusters/ztp-policies/common/group-policies/group-smcx12-vse2.yaml:168]: Given the very last changes in the linuxptp-daemon from last week looks like ts2phc will read directly from /dev/gnss0.  Let's recheck `fiesta` first before accepting this one:

```suggestion
            ts2phc.nmea_serialport /dev/gnss0
```

File: clusters/ztp-policies/common/group-policies/group-smcx12-vse2.yaml [hunk 2]
```diff
@@ -37,7 +43,133 @@ spec:
...
     #       storageType: "storage-class-http-events"
-
+    - fileName: PtpConfigGmWpc.yaml
+      policyName: "config-policy"
+      spec:
+        profile:
+        - name: "grandmaster"
+          phc2sysOpts: -r -u 0 -m -O -37 -N 8 -R 16 -s ens6f0 -n 24
+          ptp4lConf: |
+            [ens6f0]
```
Comment (jnunyez) [inline clusters/ztp-policies/common/group-policies/group-smcx12-vse2.yaml:53]: Just checking if I got this right. This means that the port acting as master port in  `falcon` and `fiesta` is always named ens6f0? Is there any other cluster in `group-smcx12`?

File: clusters/ztp-policies/common/group-policies/group-smcx12-vse2.yaml [hunk 3]
```diff
@@ -37,7 +43,133 @@ spec:
...
+            masterOnly 1
```
Comment (jnunyez) [inline clusters/ztp-policies/common/group-policies/group-smcx12-vse2.yaml:54]: Just a thought to discuss. Not urgent. Should we add all the ports in the WPC? Bc if used, all the ports will be used as master ports so they wont take any other role.

```suggestion
            masterOnly 1
            [ens6f1]
            masterOnly 1
            [ens6f2]
            masterOnly 1
            [ens6f3]
            masterOnly 1
```

**Review by jnunyez — COMMENTED** [review]
Added a few comments to discuss/try.

File: clusters/ztp-policies/common/group-policies/group-smcx12-vse2.yaml (hunk 2 above)
Comment (novacain1) [reply clusters/ztp-policies/common/group-policies/group-smcx12-vse2.yaml:53]: Right, we're creating a group based policy that's aligned to the SKU of systems i have ordered, where cards are in the same slot across.

File: clusters/ztp-policies/common/group-policies/group-smcx12-vse2.yaml (hunk 3 above)
Comment (novacain1) [reply clusters/ztp-policies/common/group-policies/group-smcx12-vse2.yaml:54]: I like this suggestion, thanks for making it here.

**Review by jnunyez — APPROVED** [review]

**Pull request merged**